Makefile

```diff
@@ -171,6 +172,10 @@ check-fast: lint $(PY35) $(PYPY) $(TOX)
 	$(TOX) -e py26-brief
 	$(TOX) -e py35-prettyquick
 
+check-rst: $(RSTLINT)
+	$(RSTLINT) CONTRIBUTING.rst
+	$(RSTLINT) README.rst
```

Comment:
We build docs with strict, which I think has the same effect. Also the docs are build with weirdo sphinx RST which has rst-lint doesn't seem to handle very well (there are apparently things that you can do to make it handle them less badly, but I think the existing docs build is probably sufficient)